### PR TITLE
(WIP) Add Encryption instructions for LocalPV and cStor volumes

### PR DESCRIPTION
**Pull request by muratkars**

### CODE REVIEW

File: docs/uglocalpv_volume_encrypt.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+## Introduction
+
+LUKS encrypted devices can be used to create OpenEBS Local PV Hostpath volumes by following the instructions. The following steps are mainly applicable for Local PV volumes running on **OpenEBS version 1.9 and above** only.
+
+## Instructions
+
+The following are the steps to provide encrypted OpenEBS Local PV volume capacity running on OpenEBS version 1.9 and above. Before you start make sure to add additional block devices to each node to be encrypted using LUKS.
+
+### Step 1: Find the new device name:
```
Comment (ranjithwingrider) [inline docs/uglocalpv_volume_encrypt.md:11]: Should we say Perform Step1 to Step 6 on each worker node where the disk is attached?

Comment (muratkars) [reply docs/uglocalpv_volume_encrypt.md:11]: Yes, this still needs some work
Missing parts:
- Key management
- Node restart
- Note to run the same on all nodes
- Complete cStor instructions

File: docs/uglocalpv_volume_encrypt.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+└─nvme0n1p2 259:3    0    128G  0 part  /
+nvme2n1     259:4    0    100G  0 disk
+└─nvme2n1p1 259:6    0     20G  0 part
+  └─backup1 254:0    0     20G  0 crypt /var/openebs/local
+```
+
+### Step 7: Create an example pod using Local PV Hostpath volume
+```
+kubectl apply -f https://openebs.github.io/charts/examples/local-hostpath/local-hostpath-pvc.yaml
+kubectl apply -f https://openebs.github.io/charts/examples/local-hostpath/local-hostpath-pod.yaml
```
Comment (ranjithwingrider) [inline docs/uglocalpv_volume_encrypt.md:189]: Also, should we mention that in there in only single device available, ensure to add nodeSelector in the application spec?

**Comment by kmova** [comment]
Taking these changes with the current content, as FAQ results in 404 without these pages.

**Pull request merged**